google/cloud/security/common/gcp_api/storage.py

```diff
@@ -16,14 +16,17 @@
 
 import StringIO
 
+from httplib2 import HttpLib2Error
+from ratelimiter import RateLimiter
+
 from google.cloud.security.common.gcp_api import _base_client
 from google.cloud.security.common.gcp_api import errors as api_errors
 from google.cloud.security.common.util import log_util
 from googleapiclient import http
+from googleapiclient.errors import HttpError
 
 LOGGER = log_util.get_logger(__name__)
 
-
```

Comment:
nit: Per the style guide, should have 2 blank lines between  top-level definitions.
```
Two blank lines between top-level definitions, one blank line between method definitions.

Two blank lines between top-level definitions, be they function or class definitions. One blank line between method definitions and between the class line and the first method. Use single blank lines as you judge appropriate within functions or methods.
```
https://google.github.io/styleguide/pyguide.html#Blank_Lines